pkg/operator/operator.go

```diff
@@ -311,12 +312,13 @@ func (o *Operator) sync(key string) error {
 		klog.Errorf("error occurred while setting status to in progress: %v", err)
 	}
 
-	err = tl.RunAll()
+	taskName, err := tl.RunAll()
 	if err != nil {
-		klog.Infof("Updating ClusterOperator status to failed. Err: %v", err)
-		reportErr := o.client.StatusReporter().SetFailed(err)
+		failedTaskReason := strings.Join(strings.Fields(taskName+"Failed"), "")
```

Comment:
Not sure if we should also convert to CamelCase here. Or let that be taken care of in CVO?

Reply:
Nvm, says in the docs:

> // reason is the reason for the condition's last transition.  Reasons are CamelCase